bin/crew

```diff
@@ -816,6 +816,14 @@ def prepare_package (destdir)
 
     # remove temporary files
     FileUtils.rm_rf ['dlistcut', '../dlist', '../filelist'], verbose: @fileutils_verbose
+
+    # Strip libraries with -S
+    strip_find_files "find . -type f -name 'lib*.a' -print", "-S"
+    strip_find_files "find . -type f -name 'lib*.so*' -print", "-S"
+
+    # Strip binaries but not compressed archives
+    strip_find_files "find . -type f ! -iname '*\.bz2' ! -iname '*\.gz' ! -iname '*\.lha' ! -iname '*\.lz' ! -iname '*\.rar' ! -iname '*\.tar' ! -iname '*\.tbz' ! -iname '*\.tgz' ! -iname '\*.txz' ! -iname '*\.xz' ! -iname '*\.zip' -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d'"
```

Comment:
Should we put these lines in a function since the same lines are called in install_package?  Or do we even need to call the lines in install_package?

Reply:
`strip_find_files` is already a function though. I think we will need a transition period to strip files at install for packages which weren't already stripped at build time, after which we can remove the install portion.

Reply:
Yes but the parameters are not.  What happens if we need to exclude another extension?  This way we would need to change in 2 places instead of just 1 if it was encapsulated in a function.  I'm just thinking like a DRY kind of guy.

Reply:
If you want to take a stab at converting that to a function, go ahead? I'm just trying to have a light touch here... I'm not sure making a function out of this would make it any faster...

Reply:
I'm not concerned about speed, only not repeating the same lines multiple times.

Reply:
How about?
```
def strip_binaries()
  extensions = [ 'bz2', 'gz', 'lha', 'lz', 'rar', 'tar', 'tbz', 'tgz', 'txz', 'xz', 'zip' ]
  inames = extensions.join(' ! -iname *\.')
  strip_find_files "find . -type f#{inames} -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d'"
end
```
Then just call `strip_binaries` in prepare_package and install_package.

Reply:
if we have .LHA files, we might also want to add .Z and .LHZ

Reply:
Thank you @saltedcoffii.  This is exactly why we need a function for this @satmandu. :)

Reply:
Done,